css-layout-classes/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Grid Classes</title>
+  </head>
+  <body>
+    <link rel="stylesheet" href="styles.css">
+    <div class="container">
+      <h1>Choose your character!</h1>
+      <div class="row">
+        <div class="column-third">
+          <img src="images/link.png" alt="link-image">
+        </div>
+        <div class="column-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Saepe accusamus impedit dolorem quis minus est error nemo ea deserunt, debitis, repudiandae vero, alias unde sapiente. Voluptatibus ad obcaecati, ullam voluptate aspernatur quae quisquam nostrum nesciunt amet praesentium iste? At, numquam!</p>
+        </div>
+        <div class="column-third">
+          <img src="images/yoshi.png" alt="yoshi-image">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-fourth">
+          <img src="images/kirby.png" alt="kirby-image">
+        </div>
+        <div class="column-fourth">
+          <img src="images/fox.png" alt="fox-image">
+        </div>
+        <div class="column-fourth">
+          <img src="images/pikachu.png" alt="pikachu-image">
+        </div>
+        <div class="column-fourth">
+          <img src="images/ness.png" alt="ness-image">
+        </div>
+      </div>
+      <h2>Donkey Kong</h2>
```

Comment:
Whoops! This should be inside the `column` element below

Reply:
So close!

Thanks for the heads up.